### PR TITLE
Create String_Pangram

### PR DESCRIPTION
**Pull request by shar2040**

### CODE REVIEW

**Review by AgentDeepak007 — CHANGES_REQUESTED** [review]
check the 4th line there is error for "raw_input"

**Comment by shar2040** [comment]
Done changes in code can you now see it

**Comment by shar2040** [comment]
Sir changed code as some error was there can you now approve

File: String_Pangram.py [hunk 1]
```diff
@@ -1,8 +1,13 @@
-from string import ascii_lowercase as asc_lower
```
Comment (AgentDeepak007) [inline String_Pangram.py]: import string before "from string import ascii_lowercase as asc_lower"

File: String_Pangram.py [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
-def check(s):
-    return set(asc_lower) - set(s.lower()) == set([])
-str=raw_input("String:")
```
Comment (AgentDeepak007) [inline String_Pangram.py]: remove "raw_input" don't copy past it from web. Your raw_input was not defined in the program any where how is it working for you.

File: String_Pangram.py [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
+def ispangram(str):
+   alphabet = "abcdefghijklmnopqrstuvwxyz"
+   for char in alphabet:
+      if char not in str.lower():
+         return False
+   return True
+
+string = 'Five boxing wizards jumping.'
+if(ispangram(string) == True):
+   print("Yes")
```
Comment (AgentDeepak007) [inline String_Pangram.py:11]: indentation error here just put one space before print("Yes")

File: String_Pangram.py [hunk 4]
```diff
@@ -1,8 +1,13 @@
...
 else:
-      print("String isn't a pangram")
```
Comment (AgentDeepak007) [inline String_Pangram.py]: indentation error in this line and below also

**Review by AgentDeepak007 — CHANGES_REQUESTED** [review]

**Comment by AgentDeepak007** [comment]
just do these corrections i will work should i send you correct code instead. Just past it and send me that........

import string

from string import ascii_lowercase as asc_lower

def check(s):
    return set(asc_lower) - set(s.lower()) == set([])
str=raw_input("String:")
if(check(str)==True):
      print("String is a pangram")
import string
def ispangram(str):
   alphabet = "abcdefghijklmnopqrstuvwxyz"
   for char in alphabet:
      if char not in str.lower():
         return False
   return True

string = 'Five boxing wizards jumping.'
if(ispangram(string) == True):
    print("Yes")
else:
    print("String isn't a pangram")
    print("No")

**Comment by shar2040** [comment]
Sir created it and sent you request now can you see

**Pull request closed without merging**